internal/dag/cache.go

```diff
@@ -358,7 +380,7 @@ func (kc *KubernetesCache) secretTriggersRebuild(secret *v1.Secret) bool {
 		}
 	}
 
-	for _, ingress := range kc.ingresses {
+	for _, ingress := range kc.ingressesv1beta1 {
```

Comment:
I think we're going to need to check both `kc.ingresses` and `kc.ingressesv1beta1`, since either one of them could be populated based on whether `v1` exists in the cluster or not.